db_stress_tool/db_stress_test_base.cc

```diff
@@ -1214,6 +1217,17 @@ void StressTest::TestCompactFiles(ThreadState* /* thread */,
   std::terminate();
 }
 #else   // ROCKSDB_LITE
+void StressTest::ClearBackupRestoreData(ThreadState* thread) {
+  std::string backup_dir = FLAGS_db + "/.backup" + ToString(thread->tid);
+  std::string restore_dir = FLAGS_db + "/.restore" + ToString(thread->tid);
+  // Reasons for this:
+  // * Free up disk space ASAP after crash & restart
+  // * Attempting to restore from backup from prior invocation could fail
```

Comment:
Just wondering, do you have an example of such an option?

Reply:
Here's the failure I'm trying to fix:

    Failure in DB::Open in backup/restore with: Not implemented: WritePrepared/WriteUnprepared txn tag when write_after_commit_ is enabled (in default WriteCommitted mode). If it is not due to corruption, the WAL must be emptied before changing the WritePolicy.

After private chat, my diagnosis is a bit wrong. We can't (necessarily) use vanilla DB::Open when blobdb or txndb are enabled. Revision coming